src/modules/subtitles.js

```diff
@@ -92,10 +92,17 @@ export default function (playerInstance, options) {
         subtitlesChangeList.style.display = 'none';
 
         let firstSubtitle = true;
+        const hasDefault = !!playerInstance.subtitlesTracks.find(track => track.default);
         playerInstance.subtitlesTracks.forEach(function (subtitle) {
+            let subtitleSelected = ''
+
+            if (subtitle.default || (!hasDefault && firstSubtitle && subtitle.label !== subtitlesOff)
+                || playerInstance.subtitlesTracks.length === 1) {
+                subtitleSelected = 'subtitle_selected'
+                playerInstance.subtitleFetchParse(subtitle);
+                firstSubtitle = false;
```

Comment:
Previously `firstSubtitle` was set outside an if statement, now it can be set as true when it's not the first subtitle track (In case the first iteration doesn't enter this if statement), maybe I'm interesting the name first subtitle wrong? If it's also the case that this needs to be set only for the first `forEach` iteration, there is no need for a variable since you can check `index === 0`.